packages/compat/package.json

```diff
@@ -14,7 +14,7 @@
     }
   },
   "files": [
-    "dist"
+    "dist/**/*.{js,cjs,mjs,d.ts}"
```

Comment:
`mjs` was not used, it's included in case there are some mjs files later.

Reply:
<img width="454" alt="Screenshot 2024-05-13 at 1 53 52 PM" src="https://github.com/eslint/rewrite/assets/32865581/9b0f748f-bda2-4dee-be4e-ce6c36eb5014">


I can see ts imports from declarations file. I dont think ignoring it would work?

Reply:
I did not see this line; have you pulled the latest changes?

Reply:
We still would need the ts file after resolving to `.js` the import inside the `d.ts` is actually resolving to `types.ts` and not `types.js`. Try looking at the compat build files once 🤔

Reply:
yes, good catch! seems it's always required (not only jsr as described in https://github.com/eslint/rewrite/pull/16#issue-2285879415).